addons/account/models/account_move.py

```diff
@@ -4403,6 +4409,46 @@ def _link_bill_origin_to_purchase_orders(self, timeout=10):
             move._find_and_set_purchase_orders(references, move.partner_id.id, move.amount_total, timeout=timeout)
         return self
 
+    def _show_autopost_bills_wizard(self):
+        if (
+            len(self) != 1
+            or self.state != "posted"
+            or not self.is_purchase_document(include_receipts=True)
+            or self.restrict_mode_hash_table
+            or all(not l.is_imported for l in self.line_ids)
+            or not self.partner_id
+            or self.partner_id.autopost_bills != "ask"
+            or not self.company_id.autopost_bills
+            or self.is_manually_modified
+        ):
+            return False
+        prev_bills_same_partner = self.search([
+            ('id', '!=', self.id),
+            ('partner_id', '=', self.partner_id.id),
+            ('state', '=', 'posted'),
+            ('move_type', 'in', self.get_purchase_types(include_receipts=True)),
+        ], order="create_date DESC", limit=10)
+        nb_unmodified = 1  # +1 for current bill that hasn't been modified either
+        for move in prev_bills_same_partner:
+            if not move.is_manually_modified:
+                nb_unmodified += 1
+            else:
+                break
+        if nb_unmodified < 3:
```

Comment:
We can't put it in the domain because of this example:
- Upload invoice1, don't modify it
- Upload invoice2, don't modify it
- Upload invoice3, modify it manually
- Upload invoice4, don't modify it

---> At this point, nb_unmodified would be equal to 3 and we would show the wizard even though we shouldn't since it's not the _last 3_ that were unmodified. The current count should be 1 unmodified, not 3, so the wizard shouldn't be shown

Reply:
Makes sense :ok_hand: